client/src/components/Home.js

```diff
@@ -6,9 +6,9 @@ const Home = props => {
   return (
     <div>
       <h1>Warm Fuzzies</h1>
-      <Link to='/'>Give</Link><br></br>
+      <Link to='/warm-fuzzies/new'>Give</Link><br></br>
```

Comment:
semantically, we should consider putting these links in an unordered list `<ul>`

Reply:
Home component removed in last PR